style-guide.md

```diff
@@ -83,9 +83,15 @@ Dla linków, które nie mają polskiego odpowiednika (Stack Overflow, filmy na Y
 
 Użyj tej sekcji, jako spisu tłumaczeń dla często używanych, technicznych pojęć.
 
-| Term   | Translation |
+| Słowo / pojęcie  | Tłumaczenie |
 | ------ | ----------- |
 | Plugin | Wtyczka     |
 | Theme  | Motyw       |
 | Query  | Zapytanie   |
 | String | Łańcuch znaków   |
+| Tutorial | Samouczek lub Poradnik   |
+| Gatsby Docs | Dokumentacja Gatsby (pisane z wielkiej litery)   |
+| Run command | Uruchom komendę   |
+| Development server | Serwer deweloperski   |
```

Comment:
Jestem za tym żeby posortować terminy wg angielskich nazw alfabetycznie. Co Ty na to?

Reply:
Dobry pomysł :) Co wy na to żeby też dodać następna tabelkę/listę pojęć, których nie będzie tłumaczyć? Typu node.js, es module itp ?